docs/blog/2018-05-31-open-sourcing-gatsby-workshops/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Open Sourcing Gatsby Workshops
+date: "2018-05-31"
+author: "Shannon Soper"
+---
+
+# Recent workshops in SF & NYC
+
+In April and May, we hosted 2-day Gatsby workshops in both San Francisco and NYC and met a lot of awesome developers. In this blogpost, we want to share what we learned and resources you can use to host your own Gatsby workshop. A big thank you to [Ryan Florence](https://twitter.com/ryanflorence) for organizing the workshop logistics; check out his company's website at [workshop.me](https://workshop.me/). Ryan has been an awesome supporter of Gatsby and a longtime fan, so yes, the workshop.me site is built on Gatsby :).
+
+# What we learned in the first few workshops
+
+Here is a photo of how many notes I took during our first workshop in San Francisco:
+
+![Photo of my workshop outline with notes scribbled all over it](workshop-notes.jpg)
+
+The biggest takeaways were:
+* People wanted to know how Gatsby works, so we drew this diagram and nearly everyone said it was helpful:
+
+![Diagram of how Gatsby build time and run time works](gatsby-diagram.JPG)
+
+* This diagram helped them understand why adding extra plugins to their Gatsby sites doesn’t slow them down (because plugins do their work at build time, not at runtime)
+* People wanted to know how Gatsby works with React and GraphQL, and we’re still working on diagrams to illustrate those concepts
+* If React and GraphQL are new to people, they seem to be the most difficult parts of learning to use Gatsby
+
+
+## Why you might want to host a workshop
+
+Here are some advantages that we experienced in the workshops; if these appeal to you, you could consider hosting your own workshop at your work, a meetup, or any other setting that makes sense:
+
+* _Dedicated time and place to learn:_ it’s easier to set aside time to learn something new when you are in a room with other people doing the same thing
+* _Quicker problem-solving:_ you can usually get answers to your questions and solve problems faster than you could otherwise
+* _Customized content:_ the workshop host can customize the workshop for the group’s needs ahead of time by sending out a survey, and the workshop can shift focus depending on real-time needs and questions of the group and individuals
+
+We'd love to hear how it goes and hope that you'll use the resources below.
+
+# Where to find workshop resources
+
+Please use the follow links to find our open source workshop materials in the Gatsby open source repo:
+* [Survey questions to send out before and after workshops]()
+* [Introductory slide deck for Day 1](www/public/static/day-1-slide-deck.pdf), [Introductory slide deck for Day 2](www/public/static/day-2-slide-deck.pdf)
```

Comment:
oh missed these — yeah the link should just be `/static/day-2-slide-deck.pdf`.

Looks like you also need to add the files to the PR

Reply:
I tried! I have no idea how...I saved everything and they just didn't show up in the Github desktop thing...I'll try again